week-4/InClass/A-objects-intro/exercise-part-1.js

```diff
@@ -6,4 +6,28 @@ can describe with a JavaScript object
 Assign each of them to a separate variable
 
 */
-
+let hoodie = {
+    brand: "Puma",
+    colours: ["purple", "black"],
+    yearsOwned: 2
+  }
+  
+  let book = {
+    author: "Daphne Du Maurier",
+    yearOfRelease = "1938",

```

Comment:
Is this the correct syntax for assigning values to your object properties? 
Also, do you want the year `1938` to be a string or a number?